src/gradleinttest.sh

```diff
@@ -53,9 +54,19 @@ gradleinttest () {
     pg_isready -h ${PGHOST} -p ${PGPORT}
     psql -U postgres -f ./db/create-data-repo-db
     # required for tests
-    ./gradlew assemble
-    ./gradlew check --scan
-    ./gradlew ${test_to_run} --scan
+    if [[ "${test_to_run}" == "testPerf" ]]; then
+      printf "perf test\n"
+      export GOOGLE_APPLICATION_CREDENTIALS=/tmp/jade-dev-account.json
+      export TEST_RUNNER_SERVER_SPECIFICATION_FILE="${NAMESPACEINUSE}.json"
```

Comment:
Is this redefining previous environment variables?

Reply:
GOOGLE_APPLICATION_CREDENTIALS? Yes, I'm just trying to get it to a working state, I have a lot of hard-coded stuff that is going to need to change.